src/ResourceManager/Profile/Commands.Profile/Account/AddAzureRmAccount.cs

```diff
@@ -34,12 +34,14 @@ namespace Microsoft.Azure.Commands.Profile
     [OutputType(typeof(PSAzureProfile))]
     public class AddAzureRMAccountCommand : AzureRMCmdlet, IModuleAssemblyInitializer
     {
-        private const string UserParameterSet = "User";
-        private const string ServicePrincipalParameterSet = "ServicePrincipal";
-        private const string ServicePrincipalCertificateParameterSet = "ServicePrincipalCertificate";
-        private const string AccessTokenParameterSet = "AccessToken";
-        private const string SubscriptionNameParameterSet = "SubscriptionName";
-        private const string SubscriptionIdParameterSet = "SubscriptionId";
+        private const string UserParameterSetWithSubscriptionId = "UserWithSubscriptionId";
```

Comment:
Need to update the DefaultParameterSetName in the Cmdlet attribute to match one of the actual parameter sets (UserWithSubscriptionId is the best bet)